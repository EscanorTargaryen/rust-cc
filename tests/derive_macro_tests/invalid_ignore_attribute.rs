use rust_cc::*;

#[derive(Trace, Finalize)]
#[rust_cc(ignore)]
struct MyStruct {
}

#[derive(Trace, Finalize)]
#[rust_cc(ignore)]
enum MyEnum {
    A(),
}

fn main() {
}
