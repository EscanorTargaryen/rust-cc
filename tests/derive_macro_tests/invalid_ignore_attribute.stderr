error: Invalid attribute position
 --> tests/derive_macro_tests/invalid_ignore_attribute.rs:4:11
  |
4 | #[rust_cc(ignore)]
  |           ^^^^^^

error: Invalid attribute position
 --> tests/derive_macro_tests/invalid_ignore_attribute.rs:9:11
  |
9 | #[rust_cc(ignore)]
  |           ^^^^^^
